driver/CMakeLists.txt

```diff
@@ -25,15 +25,17 @@
 ################################################################################
 
 find_package(Threads REQUIRED)
+find_package(rocrand REQUIRED)
+message(STATUS "rocrand_VERSION= ${rocrand_VERSION}")
 
-add_executable(MIOpenDriver main.cpp InputFlags.cpp)
+add_executable(MIOpenDriver main.cpp InputFlags.cpp rocrand_wrapper.cpp)
```

Comment:
@atamazov in the top-most CMakeLists.txt we need to make rocrand a dependency

Reply:
and ultimately make `MIOpen package` be dependent on `rocRAND`

Reply:
@junliume 
> in the top-most CMakeLists.txt

I am wondering why all this can't be done right in driver/CMakeLists.txt, but will do ;)

Reply:
@junliume Fixed in 9ada41aab14643469c55c0d00251bf36a962f89f

Reply:
@atamazov IMHO in order to fix this issue we need to add a line of `rocrand` right below this:
https://github.com/ROCm/MIOpen/blob/cdb4c740c361cc75d053d4b69463ecb2c813468e/Dockerfile#L52

Reply:
and some where near https://github.com/ROCm/MIOpen/blob/cdb4c740c361cc75d053d4b69463ecb2c813468e/CMakeLists.txt#L484-L486
```
set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocrand")
```